src/lib.rs

```diff
@@ -254,13 +254,16 @@ impl Client {
     /// On Unix and Windows this will clobber the `CARGO_MAKEFLAGS` environment
     /// variables for the child process, and on Unix this will also allow the
     /// two file descriptors for this client to be inherited to the child.
+    ///
+    /// On platforms other than Unix and Windows this does nothing.
```

Comment:
It seems like it may be more appropriate to panic on other platforms? It's not supported there so only in-process usage is expected to work

Reply:
That would make impossible to use same code compiled to both native and WASI, without extra `#[cfg(...)]`.

Reply:
@RReverser it would be possible, as WASI doesn't support spawning processes, so if you were to call `configure` you would likely panic shortly afterwards for spawning a thread anyway. Also because the function keeps existing no `#[cfg(...)]` is required to keep it compiling.